lib/diaspora/mentionable.rb

```diff
@@ -15,6 +15,7 @@ def self.mention_attrs(mention_str)
     del_pos = mention.rindex(/;/)
 
     name   = mention[0..(del_pos-1)].strip
+    name = nil if name.empty?
```

Comment:
It was done for [this line](https://github.com/diaspora/diaspora/pull/6805/files#diff-515d5bec596509fb74bd38d45f5c3db2R100) to pass to `person.name` when empty. However, it's probably better to change the mentioned line then, instead of having this non-obvious expression.